### PR TITLE
DEV: Add comments about URLs on CLI

### PR DESCRIPTION
**Pull request by gabrieljenik**
- Comments on config.php about baseUrl and publicurl (related to 18961)

### CODE REVIEW

**Review by Shnoulle — APPROVED** [review]
OI really like to have a file read and updated , more easy to construct.

File: application/controllers/InstallerController.php [hunk 1]
```diff
@@ -972,7 +977,12 @@ private function writeConfigFile()
             . "\t" . "// on your webspace." . "\n"
             . "\t" . "// LimeSurvey developers: Set this to 2 to additionally display STRICT PHP error messages and get full access to standard templates" . "\n"
             . "\t\t" . "'debug'=>0," . "\n"
-            . "\t\t" . "'debugsql'=>0, // Set this to 1 to enanble sql logging, only active when debug = 2" . "\n";
+            . "\t\t" . "'debugsql'=>0, // Set this to 1 to enanble sql logging, only active when debug = 2" . "\n"
+            . "\n"
+            . "\t\t" . "// If URLs generated while running on CLI are wrong, you need to uncomment the following line and set your" . "\n"
+            . "\t\t" . "// public URL (the URL facing survey participants). You will also need to set the request->baseUrl in the section above." . "\n"
```
Comment (Shnoulle) [inline application/controllers/InstallerController.php:983]: But ? Set only publicurl work good before in CLI too ?
No ?

Comment (gabrieljenik) [reply application/controllers/InstallerController.php:983]: What do you mean by "before"?

As to give some context, this is a followup on [18961](https://bugs.limesurvey.org/view.php?id=18961)

Comment (Shnoulle) [reply application/controllers/InstallerController.php:983]: In 3.X

**Review by Shnoulle — APPROVED** [review]

**Pull request merged**